nvflare/private/fed/app/simulator/simulator.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Federated server launching script."""
+
+import argparse
+import os
+import sys
+
+from nvflare.private.fed.app.simulator.simulator_runner import SimulatorRunner
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("job_folder")
+    parser.add_argument("--workspace", "-o", type=str, help="WORKSPACE folder", required=True)
+    parser.add_argument("--clients", "-n", type=int, help="number of clients")
```

Comment:
The client name list option is used in priority over the number of clients option. When it's provided, it will be used as the simulated client name list.